package.json

```diff
@@ -51,8 +51,10 @@
     "@angular/platform-browser-dynamic": "2.0.0",
     "@angular/platform-server": "2.0.0",
     "@angular/router": "3.0.0",
+    "@angular2-material/core": "^2.0.0-alpha.8-2",
```

Comment:
Please remove all material-design/bootstrap stuff for now. Will return to this later.